### PR TITLE
[OV] Add SoftMax -> Convert to ignored patterns

### PR DESCRIPTION
**Pull request by l-bat**
### Changes

Add Softmax -> Convert -> MatMul to `IgnoredPatternNames.MULTIHEAD_ATTENTION_OUTPUT`

### Reason for changes

To remove FQ on attention mask
image

### Ticket
117617

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
Could you provide a model for which this applies?
